modules/analytics/src/views/full/index.tsx

```diff
@@ -444,18 +460,41 @@ const Analytics: FC<any> = ({ bp }) => {
     link.click()
   }
 
+  const readFile = (e: any) => {
```

Comment:
what's that filereader for ? we allow to import analytics  ?

Reply:
Just loads data in the UI, no import feature